tests/php/sync/test_class.jetpack-sync-full.php

```diff
@@ -825,6 +825,18 @@ function test_full_sync_doesnt_send_deleted_users() {
 		$this->assertEquals( $keep_user_id, $users[ $existing_user_count ]->ID );
 	}
 
+	function test_full_sync_doesnt_exceed_options_write_rate_limit() {
+		foreach( range( 1, 2100 ) as $i ) { // 200 items+
```

Comment:
good call - let's do that in a separate PR otherwise we'll have to wait for the tests to run again :/